src/Promitor.Scraper.Tests.Unit/Serialization/MetricsDeclaration/MetricsDeclarationWithAzureStorageQueueYamlSerializationTests.cs

```diff
@@ -15,13 +16,15 @@ namespace Promitor.Scraper.Tests.Unit.Serialization.MetricsDeclaration
     [Category("Unit")]
     public class MetricsDeclarationWithAzureStorageQueueYamlSerializationTests : YamlSerializationTests<StorageQueueMetricDefinition>
     {
-        [Fact]
-        public void YamlSerialization_SerializeAndDeserializeValidConfigForAzureStorageQueue_SucceedsWithIdenticalOutput()
+        [Theory]
+        [InlineData(@"01:00", @"2:00")]
```

Comment:
Would it make sense to add a new `[InlineData(@"01:00", null)]` to cover default, but no override?

Reply:
I actually considered making all 4 combinations after seeing xUnit didn't do it inherently and being greatly disappointed... happy to make that change.

Reply:
I think I'm going to punt on this since every param would result in 2^n permutations needed... figured let's make sure we can handle values and nulls in all parameters